iati/home/management/commands/updatestatistics.py

```diff
@@ -0,0 +1,39 @@
+import requests
```

Comment:
I would explicitly add `requests` to `requirements.txt` as it is an explicit dependency.

Reply:
Resolved in 8bbaa86